shopgate/pwa
# Implemented the process type `PROCESS_LAST` in the `PipelineManager` and changed the `fetchCart` action accordingly. `fetchCart` now returns a promise.

# Description

Implemented the process type "last" in the pipeline manager and changed the `fetchCart` action to utilise this new type. It now also returns a promise that can be waited for.

## Type of change

Please add an "x" into the option that is relevant:


## How to test it

Try sending out a duplicate `getCart` request and observe only one single "RECEIVE_CART" action to be triggered with the latest data.
